### PR TITLE
Install Miniconda3-4.5.4 (Python 3.6) on Windows

### PR DESCRIPTION
**Pull request by tkf**
This is a workaround for:

* https://github.com/JuliaLang/IJulia.jl/issues/739
* https://github.com/ContinuumIO/anaconda-issues/issues/10082
* https://github.com/JuliaPy/PyCall.jl/pull/556#issuecomment-421514385
* https://github.com/JuliaPy/PyCall.jl/issues/575

### CODE REVIEW

**Comment by tkf** [comment]
There is one failure in AppVeyor due to network problem https://ci.appveyor.com/project/StevenGJohnson/conda-jl/build/1.0.125

Except for that, it looks like it's working.

File: src/Conda.jl [hunk 1]
```diff
@@ -128,6 +128,13 @@ function _installer_url()
     elseif Compat.Sys.islinux()
         res *= "Linux"
     elseif Compat.Sys.iswindows()
+        if MINICONDA_VERSION == "3"
+            # Quick fix for:
+            # * https://github.com/JuliaLang/IJulia.jl/issues/739
+            # * https://github.com/ContinuumIO/anaconda-issues/issues/10082
+            # * https://github.com/conda/conda/issues/7789
+            res = "https://repo.continuum.io/miniconda/Miniconda$(MINICONDA_VERSION)-4.5.4-"
```
Comment (tkf) [inline src/Conda.jl]: The main part of this PR is this `if` block. Everything else is test/CI tweaks.

**Comment by tkf** [comment]
It works with PyCall: https://github.com/JuliaPy/PyCall.jl/pull/577

**Comment by stevengj** [comment]
Okay. We can just take out this special case when the issue is fixed upstream. (Soon, I hope?)

**Comment by tkf** [comment]
I hope so. Let's track it in #125.

**Comment by stevengj** [comment]
I forgot we did this!   It's been two years, probably time to revert this patch!

In particular, it seems like Python 3.6 with breaks Tornado on Windows these days.

**Pull request merged**